packages/mux-audio/src/index.ts

```diff
@@ -6,13 +6,15 @@ import {
   MuxMediaProps,
   StreamTypes,
   ValueOf,
-  ExtensionMimeTypeMap,
   toMuxVideoURL,
   teardown,
-  PlaybackEngine,
   Metadata,
   mux,
-  type UpdateAutoplay,
+} from "@mux-elements/playback-core";
+import type {
```

Comment:
question(non-blocking): why move the type imports into a separate declaration?

Reply:
web-test-runner was complaining about this. not sure why but this fixed it

Reply:
wonder if it's something like an esbuild version mismatch and they're using an older esbuild than we do.